internal/cluster/cluster.go

```diff
@@ -571,3 +576,40 @@ func (m *Manager) IsReadyForInstallation(c *common.Cluster) (bool, string) {
 	}
 	return true, ""
 }
+
+func (m *Manager) SetConnectivityMajorityGroupsForCluster(c *common.Cluster, db *gorm.DB) error {
+	if db == nil {
+		db = m.db
+	}
+	if !funk.ContainsString([]string{models.ClusterStatusPendingForInput, models.ClusterStatusInsufficient, models.ClusterStatusReady}, swag.StringValue(c.Status)) {
```

Comment:
Please add a why comment - it's not clear when reading the code.

Reply:
OK